aws-lambda-java-events-sdk-transformer/src/test/java/com/amazonaws/services/lambda/runtime/events/transformers/dynamodb/DynamodbStreamRecordTransformerTest.java

```diff
@@ -101,4 +101,24 @@ public void testToStreamRecordV2() {
         StreamRecord convertedStreamRecord = DynamodbStreamRecordTransformer.toStreamRecordV2(streamRecord_event);
         Assertions.assertEquals(streamRecord_v2, convertedStreamRecord);
     }
+
+    @Test
+    public void testToStreamRecordV2WhenOldImageIsNull() {
+        com.amazonaws.services.lambda.runtime.events.models.dynamodb.StreamRecord streamRecord = streamRecord_event.clone();
+        streamRecord.setOldImage(null);
+
+        Assertions.assertDoesNotThrow(() -> {
+            StreamRecord convertedStreamRecord = DynamodbStreamRecordTransformer.toStreamRecordV2(streamRecord);
```

Comment:
No need to store the return value here - you could just write it as:
```java
Assertions.assertDoesNotThrow(() -> {
    DynamodbStreamRecordTransformer.toStreamRecordV2(streamRecord);
});
```

Reply:
Addressed